packages/shared/src/forms/datePicker/DatePicker.ios.js

```diff
@@ -36,7 +40,9 @@ export default class DatePicker extends React.Component<Props, State> {
   };
 
   onSave = () => {
-    this.props.onDateChange(this.state.date);
+    // if this.state.date is null, fallback sent to DatepickerIOS is new Date()
```

Comment:
These comments aren't completely accurate anymore since we are using `DateUtils.getUTCToday()`, so maybe just correct this.